x-pack/plugins/code/public/components/main/side_tabs.tsx

```diff
@@ -95,6 +96,7 @@ class CodeSideTabs extends React.PureComponent<Props> {
           initialSelectedTab={this.tabs.find(t => t.id === this.sideTab)}
           onTabClick={tab => this.switchTab(tab.id as Tabs)}
           expand={true}
+          selectedTab={this.tabs.find(t => t.id === this.sideTab)}
```

Comment:
@mw-ding Code intelligence doesn't carry over tab in the url, but we have to control the `selectedTab` here.

Reply:
what do you mean by code intelligence? We want to make sure if the `tab=structure` param will be carried over to a new page by jumpping to reference/definition.